diode-server/reconciler/logs_retriever.go

```diff
@@ -0,0 +1,169 @@
+package reconciler
+
+import (
+	"bytes"
+	"context"
+	"encoding/base64"
+	"encoding/binary"
+	"encoding/json"
+	"fmt"
+	"strconv"
+
+	"github.com/netboxlabs/diode/diode-server/gen/diode/v1/reconcilerpb"
+)
+
+type extraAttributesWrapper struct {
+	ExtraAttributes string `json:"$"`
+	IngestionTs     string `json:"ingestion_ts"`
+}
+type redisLogResult struct {
+	ExtraAttributes extraAttributesWrapper `json:"extra_attributes"`
+	ID              string                 `json:"id"`
+	Values          []interface{}          `json:"values"`
```

Comment:
Do we need this at all?

Reply:
I don't think so, I will remove it as well. I've already removed some unused fields

Reply:
Probably we can remove from test inputs as well

Reply:
I kept on log to mock the full data from the redis FT SEARCH request